packages/kit/src/runtime/server/page/render.js

```diff
@@ -353,17 +353,32 @@ export async function render_response({
 	// add the content after the script/css links so the link elements are parsed first
 	head += rendered.head;
 
+	const html = options.app_template({
+		head,
+		body,
+		assets,
+		nonce: /** @type {string} */ (csp.nonce)
+	});
+
 	// TODO flush chunks as early as we can
-	const html =
+	const transformed =
 		(await resolve_opts.transformPageChunk({
-			html: options.app_template({ head, body, assets, nonce: /** @type {string} */ (csp.nonce) }),
+			html,
 			done: true
 		})) || '';
 
+	if (__SVELTEKIT_DEV__) {
```

Comment:
I think you could just `import { DEV } from 'esm-env'` rather than introducing `__SVELTEKIT_DEV__`

Reply:
ah, of course! totally forgot. much nicer